app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/activity/BuyTicketActivity.java

```diff
@@ -204,6 +204,8 @@ private void handleTicketReservationSuccess(TicketType ticketType,
         Intent intent = new Intent(this, StripePaymentActivity.class);
         intent.putExtra(Const.KEY_TICKET_PRICE, ticketType.getFormattedPrice());
         intent.putExtra(Const.KEY_TICKET_HISTORY, response.getTicketHistory());
+        intent.putExtra(Const.KEY_TERMS_LINK, ticketType.getPaymentInfo().getTermsLink());
+        intent.putExtra(Const.KEY_STRIPE_API_PUBLISHABLE_KEY, ticketType.getPaymentInfo().getStripePublicKey());
```

Comment:
I mean it should not happen, but what happens if paymentInfo is `null`

Reply:
@kordianbruck paymentInfo is a not-null type in Kotlin.